### PR TITLE
E2E GitHub action

### LINKED ISSUE #30: Testing the extension (GitHub action)
Introduce (where applicable) unit and integration (Playwright) testing. Ideally, we would verify:

- filter cards, giftcards, ibans
- save as favourites
- copy in clipboard

It doesn't seem possible to use Playwright to test the interaction of the extension with the Drop-in component. Investigate further to assess the feasibility or other options.

### PR DESCRIPTION
**Pull request by gcatanese**
Create GitHub Action to run E2E tests

Fix #30

### CODE REVIEW

File: .github/workflows/e2e.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+name: E2E Testing
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ main ]
+    paths-ignore:
+      - '**/README.md'
```
Comment (Kwok-he-Chu) [inline .github/workflows/e2e.yml:8]: Should be README.md

File: .github/workflows/e2e.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      - .gitignore
+      - LICENSE
+  pull_request:
+    branches: [ main ]
+    paths-ignore:
+      - '**/README.md'
```
Comment (Kwok-he-Chu) [inline .github/workflows/e2e.yml:14]: Should be README.md

**Review by Kwok-he-Chu — APPROVED** [review]
Minor change

I've noticed a warning produced when `Run actions/upload-artifact@v3` is executed, check whether this is intended

> Warn: "No files were found with the provided path: playwright-report/. No artifacts will be uploaded."

**Comment by gcatanese** [comment]
> Minor change
> 
> I've noticed a warning produced when `Run actions/upload-artifact@v3` is executed, check whether this is intended
> 
> > Warn: "No files were found with the provided path: playwright-report/. No artifacts will be uploaded."

I have removed this step

**Pull request merged**